### ISSUE TITLE
Move utility functions to src/utils.ts

### ISSUE THREAD

**Issue by campionfellin** [google/clasp#133]
This will improve the readability and maintainability of `clasp`

* [ ] Move util objects and functions like DOT, DOTFILE, etc to `src/utils.ts`
* [ ] Integrate changes into `index.ts`
* [ ] Run all the tests (all are relevant)

**Comment by grant** [comment]
SGTM.
I'd probably do this first before moving the commands and make sure the `tsc` command is correct.

**Comment by campionfellin** [comment]
Yeah this issue is especially important to improving our test coverage. Once separated, it should be much easier to test methods like: https://github.com/google/clasp/blob/9ad0a4220b6b84b9a9eb6a511297e24461e2dae5/index.ts#L263
